sechub-cli/src/mercedes-benz.com/sechub/cli/constants.go

```diff
@@ -17,13 +19,25 @@ const DefaultTempDir = "."
 
 // DefaultWaitTime - Wait time in seconds.
 // Will be used
-// - for automatic status checks etc. when action=scan
+// - for maximum pause between automatic status checks when action=scan
 // - for pause between retries for failed HTTP calls
 const DefaultWaitTime = 60
 
+// MinimalWaitTimeSeconds - We don't allow intervals shorter than this to protect the SecHub server
+const MinimalWaitTimeSeconds = 4
+
+// InitialWaitIntervallSeconds WaitIntervallIncreaseFactor - defines client's polling behaviour:
+// 2s - 3s - 4.5s - 7s - 10s - 15s - 23s - 34s - 51s - 60s - 60s - 60s ...
```

Comment:
Is 4.5s really possible? Casting something to `int64` should return an int value. In addition, why not using a uint64 data type?

Reply:
Yes: 4.5s is possible. It's converted into nanoseconds.
The `time` functions expect `int64`, so we use it here also.